PencilCase/PencilCase/scripts/Navigator.ts

```diff
@@ -24,8 +25,13 @@ export class Navigator {
     // the navigation only works with JQuery object, it must be due to the navigation ways I implemented.
     // So I restraint the type to be JQuery object here to avoid spending time on debugging.
     // Whatever, I don't want to spend more time on it since it already took me much time to make everything works as it does currently
-    public navigateTo = (toPage: JQuery, options?: any) => {
-        (<any>($)).mobile.changePage(toPage, options);
+    public navigateTo = (pageInfo: any, options?: any) => {
```

Comment:
Refactored the navigateTo to make calling easier, with less parameters and avoid typos.